### PR TITLE
Echo Test: Use 17171 as the port for probes for custom grpc app

### PR DESCRIPTION
**Pull request by yashykt**
Use 17171 as the port for probes for the custom grpc app, since 3333 would be listened on by the Go server

### CODE REVIEW

File: common/scripts/kind_provisioner.sh [hunk 1]
```diff
@@ -416,7 +416,7 @@ function cidr_to_ips() {
     python3 - <<EOF
 from ipaddress import ip_network;
 from itertools import islice;
-[print(str(ip) + "/" + str(ip.max_prefixlen)) for ip in islice(ip_network('$CIDR').hosts(), 1000, 1100)]
+[print(str(ip) + "/" + str(ip.max_prefixlen)) for ip in islice(ip_network('$CIDR').hosts(), 200, 1100)]
```
Comment (stevenctl) [inline common/scripts/kind_provisioner.sh:419]: when merging common-files changes, you have to go through the github.com/istio/common-files repo

File: pkg/test/framework/components/echo/kube/deployment.go [hunk 2]
```diff
@@ -263,14 +263,14 @@ spec:
           failureThreshold: 10
         livenessProbe:
           tcpSocket:
-            port: tcp-health-port
+            port: 3333
```
Comment (stevenctl) [inline pkg/test/framework/components/echo/kube/deployment.go:266]: I think we should continue to use port naming for this, and change the portion of the template near line 234 where we say "if port == 3333 { this is the health port}

File: common/scripts/kind_provisioner.sh (hunk 1 above)
Comment (yashykt) [reply common/scripts/kind_provisioner.sh]: reverted

File: pkg/test/framework/components/echo/kube/deployment.go (hunk 2 above)
Comment (yashykt) [reply pkg/test/framework/components/echo/kube/deployment.go]: done

**Review by stevenctl — APPROVED** [review]

**Review by costinm — APPROVED** [review]

**Pull request merged**